packages/component-library/src/LandingPage/canvas-code.js

```diff
@@ -0,0 +1,199 @@
+	// of the center point of our arc as x and y coordinates.
+	// The third value is for radius, then start angle,
+	// end angle and finally a boolean value which decides
+	// whether the arc is to be drawn in counter clockwise or
+	// in a clockwise direction. False for clockwise.
+	this.draw = function() {
+		ctx.fillStyle = "white";
+		ctx.beginPath();
+		ctx.arc(this.x, this.y, this.radius, 0, Math.PI * 2, false);
+
+		// Fill the color to the arc that we just created
+		ctx.fill();
+	}
+}
+
+// Time to push the particles into an array
+for(var i = 0; i < particleCount; i++) {
+	particles.push(new Particle());
+}
+
+// Function to draw everything on the canvas that we'll use when
+// animating the whole scene.
+function draw() {
+
+	// Call the paintCanvas function here so that our canvas
+	// will get re-painted in each next frame
+	paintCanvas();
+
+	// Call the function that will draw the balls using a loop
+	for (var i = 0; i < particles.length; i++) {
+		p = particles[i];
+		p.draw();
+	}
+
+	//Finally call the update function
+	update();
+}
+
+// Give every particle some life
+function update() {
+
+	// In this function, we are first going to update every
+	// particle's position according to their velocities
+	for (var i = 0; i < particles.length; i++) {
+		p = particles[i];
+
+		// Change the velocities
+		p.x += p.vx;
+		p.y += p.vy
+
+		// We don't want to make the particles leave the
+		// area, so just change their position when they
+		// touch the walls of the window
+		if(p.x + p.radius > W)
+			p.x = p.radius;
+
+		else if(p.x - p.radius < 0) {
+			p.x = W - p.radius;
+		}
+
+		if(p.y + p.radius > H)
+			p.y = p.radius;
+
+		else if(p.y - p.radius < 0) {
+			p.y = H - p.radius;
+		}
+
+		// Now we need to make them attract each other
+		// so first, we'll check the distance between
+		// them and compare it to the minDist we have
+		// already set
+
+		// We will need another loop so that each
+		// particle can be compared to every other particle
+		// except itself
+		for(var j = i + 1; j < particles.length; j++) {
+			p2 = particles[j];
+			distance(p, p2);
+		}
+
+	}
+}
+
+// Distance calculator between two particles
+function distance(p1, p2) {
+	var dist,
+		dx = p1.x - p2.x;
+		dy = p1.y - p2.y;
+
+	dist = Math.sqrt(dx*dx + dy*dy);
+
+	// Draw the line when distance is smaller
+	// then the minimum distance
+	if(dist <= minDist) {
+
+		// Draw the line
+		ctx.beginPath();
+		ctx.strokeStyle = "rgba(255,255,255,"+ (1.2-dist/minDist) +")";
+		ctx.moveTo(p1.x, p1.y);
+		ctx.lineTo(p2.x, p2.y);
+		ctx.stroke();
+		ctx.closePath();
+
+		// Some acceleration for the partcles
+		// depending upon their distance
+		var ax = dx/200000,
+			ay = dy/200000;
+
+		// Apply the acceleration on the particles
+		p1.vx -= ax;
+		p1.vy -= ay;
+
+		p2.vx += ax;
+		p2.vy += ay;
+	}
+}
+
+// Start the main animation loop using requestAnimFrame
+function animloop() {
+	draw();
+	requestAnimFrame(animloop);
+}
+
+animloop();
```

Comment:
Is this still necessary? Looks to be a duplicate of what is now a component.

Reply:
Good catch! This file is redundant and not needed anymore. Removing.